requirements/requirements_ansible.txt

```diff
@@ -4,124 +4,130 @@
 azure-mgmt-batch==4.1.0
 azure-mgmt-compute==2.1.0
-azure-mgmt-containerservice==1.0.0
-azure-mgmt-dns==1.0.1
+azure-mgmt-containerservice==3.0.1
+azure-mgmt-dns==1.2.0
 azure-mgmt-keyvault==0.40.0
-azure-mgmt-network==1.4.0
-azure-mgmt-nspkg==2.0.0   # via azure-mgmt-batch, azure-mgmt-compute, azure-mgmt-containerservice, azure-mgmt-dns, azure-mgmt-keyvault, azu
-azure-mgmt-resource==1.1.0
+azure-mgmt-network==1.7.1
+azure-mgmt-nspkg==2.0.0   # via azure-mgmt-batch, azure-mgmt-compute, azure-mgmt-containerservice, azure-mgmt-dns, azure-mgmt-keyvault, azure-mgmt-network, azure-mgmt-resource, azure-mgmt-sql, azure-mgmt-storage, azure-mgmt-web
+azure-mgmt-resource==1.2.2
 azure-mgmt-sql==0.7.1
-azure-mgmt-storage==1.2.1
+azure-mgmt-storage==1.5.0
 azure-mgmt-web==0.32.0
 azure-nspkg==2.0.0        # via azure-cli-nspkg, azure-common, azure-mgmt-nspkg, azure-storage
 azure-storage==0.35.1
-babel==2.3.4              # via osc-lib, oslo.i18n, python-cinderclient, python-glanceclient, python-neutronclient, python-novaclient, python-openstackclient
+babel==2.5.3              # via osc-lib, oslo.i18n, python-cinderclient, python-neutronclient, python-novaclient
 backports.ssl-match-hostname==3.5.0.1
+bcrypt==3.1.4             # via paramiko
 boto3==1.4.4
 boto==2.46.1
-botocore==1.5.72          # via boto3, s3transfer
-certifi==2017.4.17        # via msrest
-cffi==1.10.0              # via cryptography
-cliff==2.7.0              # via osc-lib, python-designateclient, python-neutronclient, python-openstackclient
-cmd2==0.7.2               # via cliff
-colorama==0.3.9           # via azure-cli-core
-cryptography==2.0.3       # via adal, azure-storage, paramiko, pyopenssl, secretstorage
-debtcollector==1.15.0     # via oslo.config, oslo.utils, python-designateclient, python-keystoneclient, python-neutronclient
-decorator==4.0.11         # via shade
-deprecation==1.0.1        # via openstacksdk
-#docutils==0.12            # via botocore
-dogpile.cache==0.6.3      # via python-ironicclient, shade
-enum34==1.1.6             # via cryptography, msrest, ovirt-engine-sdk-python
+botocore==1.5.95          # via boto3, s3transfer
+certifi==2018.1.18        # via msrest
+cffi==1.11.4              # via bcrypt, cryptography, pynacl
+cliff==2.11.0             # via osc-lib, python-designateclient, python-neutronclient
+cmd2==0.8.0               # via cliff
+colorama==0.3.9           # via azure-cli-core, knack
+contextlib2==0.5.5        # via cmd2
+cryptography==2.1.4       # via adal, azure-storage, paramiko, pyopenssl, requests-kerberos, requests-ntlm, secretstorage
+debtcollector==1.19.0     # via oslo.config, oslo.utils, python-designateclient, python-keystoneclient, python-neutronclient
+decorator==4.2.1          # via openstacksdk, shade
+deprecation==1.1          # via openstacksdk
+docutils==0.14            # via botocore
```

Comment:
The `docutils` line needs to be commented, since it will be installed instead from `requirements_ansible_git.txt`.

Reply:
Thanks. Will do!

Reply:
Hey @cchurch, I fixed the issue, but GitHub says, that change is still requested. Am I missing something?